packages/analytics-js-integrations/src/integrations/Amplitude/utils.js

```diff
@@ -50,4 +50,18 @@ const getFieldsToUnset = integrations => {
   }
   return undefined;
 };
-export { getTraitsToSetOnce, getTraitsToIncrement, getDestinationOptions, getFieldsToUnset };
+
+/**
+ * Formats the given URL by adding the "https://" prefix if it doesn't already have it.
+ * 
+ * @param {string} url - The URL to be formatted.
+ * @returns {string} - The formatted URL.
+ */
+function formatUrl(url) {
+  if (url.startsWith('http://') || url.startsWith('https://')) {
```

Comment:
should we even allow http:// (unsecured) endpoint?

Reply:
yes you are right. Removing scope for http

Reply:
Is it not possible by the customer to not set up a secure tunnel?